app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  after_save :update_likes_counter
+
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_likes_counter` should be marked as private after implementing the `after_save` method.